apps/store/src/blocks/AccordionBlock.tsx

```diff
@@ -18,19 +19,30 @@ type Props = SbBaseBlockProps<{
 }>
 
 export const AccordionBlock = ({ blok, nested }: Props) => {
+  const [openedItem, setOpenedItem] = useState<string>()
```

Comment:
nit: naming, shouldn't be past tense right?
```suggestion
  const [openItem, setOpenItem] = useState<string>()
```

Reply:
🤔 I've followed the same "pattern" from [`Perils`](https://github.com/HedvigInsurance/racoon/blob/main/apps/store/src/components/Perils/Perils.tsx#L70).
Maybe I should change there as well.